### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by gwenchee**
this pr addresses https://github.com/arfc/d3ploy/issues/218

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,4 +10,9 @@ RUN conda install pyqt -y && \
     pip install -U pytest && \
     pip uninstall h5py && \
     pip install --no-cache-dir h5py && \
-    python setup.py install
+    python setup.py install && \
+    sudo apt-get update && \
```
Comment (FlanFlanagan) [inline Dockerfile:14]: Why update so many times?

**Review by FlanFlanagan — DISMISSED** [review]

File: Dockerfile (hunk 1 above)
Comment (gwenchee) [reply Dockerfile:14]: Not sure. That's what the person did on stackoverflow (https://stackoverflow.com/questions/48965309/h5py-import-error-on-libhdf5-serial-so-100)

Comment (FlanFlanagan) [reply Dockerfile:14]: interesting.

**Comment by FlanFlanagan** [comment]
My local tests passed as expected with my PR. Are yours passing with this one?

**Comment by gwenchee** [comment]
The local tests were passing prior to your PR, I think the issue is with hdf5 when using the circle ci.

**Pull request merged**